packages/create-redwood-app/template/web/package.json

```diff
@@ -17,7 +17,7 @@
     "@redwoodjs/router": "3.2.0",
     "@redwoodjs/web": "3.2.0",
     "prop-types": "15.8.1",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
```

Comment:
Kind of feels like we shouldn't be pinning these in the CRWA template; let me see what that involves.